types/express-serve-static-core/index.d.ts

```diff
@@ -447,7 +460,7 @@ export interface Request<P extends Params = ParamsDictionary> extends http.Incom
     xhr: boolean;
 
     //body: { username: string; password: string; remember: boolean; title: string; };
-    body: any;
+    body: { [key: string]: string | undefined };
```

Comment:
What if body comes with nested object?
```
{
   user: { name: string },
   password: string,
}
```

Reply:
@kacepe , not a problem, then if that is the concern, I could refactor my PR like so:

`body: { user: { name: string; }, password: string; }`